src/OAuth/AccessKey.cs

```diff
@@ -20,5 +22,14 @@ public class AccessKey
 
         [JsonProperty("jwk")]
         public JsonWebKey Jwk { set; get; }
+
+        private static AccessKey accessKey;
```

Comment:
is this variable needed?

Reply:
i guess not, i can just use the local variable inside the function instead